### PR TITLE
Bump Qt version to 5.14.1

### PR DESCRIPTION
**Pull request by m-kuhn**
Rebased version of #149

- [ ] Build QtWebEngine
  - Fix installation of Python 2
  - or upgrade QtWebEngine build engine for Python 3 (Python 2 is EOL)

- [ ] Get more space for windows builds

```
..\..\include\QtMultimedia\QtMultimediaDepends(7): fatal error C1085: Cannot write precompiled header file: '.pch\release\Qt5Multimedia_conda_pch.pch': There is not enough space on the disk.
```

- [ ] Patch macOS specific code

```
$BUILD_PREFIX/bin/x86_64-apple-darwin13.4.0-clang++ -c -include.pch/Qt5Gui_x86_64.pch/c++_x86_64 -pipe -stdlib=libc++ -Oz -fomit-frame-pointer -fdata-sections -fvisibility=hidden -std=c++1y -fapplication-extension  -arch x86_64 -isysroot /Applications/Xcode_11.3.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk -mmacosx-version-min=10.13 -fvisibility=hidden -fvisibility-inlines-hidden -fno-exceptions -Wall -Wextra -Winconsistent-missing-override -Wobjc-interface-ivars -Wobjc-method-access -Wobjc-multiple-method-names -Werror=unguarded-availability -Werror=unguarded-availability-new -Werror=unsupported-availability-guard -fPIC -DQT_NO_LINKED_LIST -DQT_NO_JAVA_STYLE_ITERATORS -DQT_NO_USING_NAMESPACE -DQT_NO_FOREACH -DENABLE_PIXMAN_DRAWHELPERS -DQT_NO_NARROWING_CONVERSIONS_IN_CONNECT -DQT_BUILD_GUI_LIB -DQT_BUILDING_QT -DQT_NO_CAST_TO_ASCII -DQT_ASCII_CAST_WARNINGS -DQT_MOC_COMPAT -DQT_USE_QSTRINGBUILDER -DQT_DEPRECATED_WARNINGS -DQT_DISABLE_DEPRECATED_BEFORE=0x050000 -DQT_DEPRECATED_WARNINGS_SINCE=0x060000 -DQT_NO_EXCEPTIONS -D_LARGEFILE64_SOURCE -D_LARGEFILE_SOURCE -DQT_NO_DEBUG -DQT_CORE_LIB -I. -I../3rdparty/md4c -I../3rdparty/VulkanMemoryAllocator -I../../include -I../../include/QtGui -I../../include/QtGui/5.14.1 -I../../include/QtGui/5.14.1/QtGui -I.tracegen -I/System/Library/Frameworks/OpenGL.framework/Headers -I/System/Library/Frameworks/AGL.framework/Headers/ -I../../include/QtCore/5.14.1 -I../../include/QtCore/5.14.1/QtCore -I../../include/QtCore -I.moc -I$PREFIX/include -I../../mkspecs/macx-clang -o .obj/qshaderdescription.o rhi/qshaderdescription.cpp
$BUILD_PREFIX/bin/x86_64-apple-darwin13.4.0-clang++ -c -include.pch/Qt5Gui_x86_64.pch/c++_x86_64 -pipe -stdlib=libc++ -Oz -fomit-frame-pointer -fdata-sections -fvisibility=hidden -std=c++1y -fapplication-extension  -arch x86_64 -isysroot /Applications/Xcode_11.3.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk -mmacosx-version-min=10.13 -fvisibility=hidden -fvisibility-inlines-hidden -fno-exceptions -Wall -Wextra -Winconsistent-missing-override -Wobjc-interface-ivars -Wobjc-method-access -Wobjc-multiple-method-names -Werror=unguarded-availability -Werror=unguarded-availability-new -Werror=unsupported-availability-guard -fPIC -DQT_NO_LINKED_LIST -DQT_NO_JAVA_STYLE_ITERATORS -DQT_NO_USING_NAMESPACE -DQT_NO_FOREACH -DENABLE_PIXMAN_DRAWHELPERS -DQT_NO_NARROWING_CONVERSIONS_IN_CONNECT -DQT_BUILD_GUI_LIB -DQT_BUILDING_QT -DQT_NO_CAST_TO_ASCII -DQT_ASCII_CAST_WARNINGS -DQT_MOC_COMPAT -DQT_USE_QSTRINGBUILDER -DQT_DEPRECATED_WARNINGS -DQT_DISABLE_DEPRECATED_BEFORE=0x050000 -DQT_DEPRECATED_WARNINGS_SINCE=0x060000 -DQT_NO_EXCEPTIONS -D_LARGEFILE64_SOURCE -D_LARGEFILE_SOURCE -DQT_NO_DEBUG -DQT_CORE_LIB -I. -I../3rdparty/md4c -I../3rdparty/VulkanMemoryAllocator -I../../include -I../../include/QtGui -I../../include/QtGui/5.14.1 -I../../include/QtGui/5.14.1/QtGui -I.tracegen -I/System/Library/Frameworks/OpenGL.framework/Headers -I/System/Library/Frameworks/AGL.framework/Headers/ -I../../include/QtCore/5.14.1 -I../../include/QtCore/5.14.1/QtCore -I../../include/QtCore -I.moc -I$PREFIX/include -I../../mkspecs/macx-clang -o .obj/qshader.o rhi/qshader.cpp
$BUILD_PREFIX/bin/x86_64-apple-darwin13.4.0-clang++ -c -include.pch/Qt5Gui_x86_64.pch/c++_x86_64 -pipe -stdlib=libc++ -Oz -fomit-frame-pointer -fdata-sections -fvisibility=hidden -std=c++1y -fapplication-extension  -arch x86_64 -isysroot /Applications/Xcode_11.3.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk -mmacosx-version-min=10.13 -fvisibility=hidden -fvisibility-inlines-hidden -fno-exceptions -Wall -Wextra -Winconsistent-missing-override -Wobjc-interface-ivars -Wobjc-method-access -Wobjc-multiple-method-names -Werror=unguarded-availability -Werror=unguarded-availability-new -Werror=unsupported-availability-guard -fPIC -DQT_NO_LINKED_LIST -DQT_NO_JAVA_STYLE_ITERATORS -DQT_NO_USING_NAMESPACE -DQT_NO_FOREACH -DENABLE_PIXMAN_DRAWHELPERS -DQT_NO_NARROWING_CONVERSIONS_IN_CONNECT -DQT_BUILD_GUI_LIB -DQT_BUILDING_QT -DQT_NO_CAST_TO_ASCII -DQT_ASCII_CAST_WARNINGS -DQT_MOC_COMPAT -DQT_USE_QSTRINGBUILDER -DQT_DEPRECATED_WARNINGS -DQT_DISABLE_DEPRECATED_BEFORE=0x050000 -DQT_DEPRECATED_WARNINGS_SINCE=0x060000 -DQT_NO_EXCEPTIONS -D_LARGEFILE64_SOURCE -D_LARGEFILE_SOURCE -DQT_NO_DEBUG -DQT_CORE_LIB -I. -I../3rdparty/md4c -I../3rdparty/VulkanMemoryAllocator -I../../include -I../../include/QtGui -I../../include/QtGui/5.14.1 -I../../include/QtGui/5.14.1/QtGui -I.tracegen -I/System/Library/Frameworks/OpenGL.framework/Headers -I/System/Library/Frameworks/AGL.framework/Headers/ -I../../include/QtCore/5.14.1 -I../../include/QtCore/5.14.1/QtCore -I../../include/QtCore -I.moc -I$PREFIX/include -I../../mkspecs/macx-clang -o .obj/qrhigles2.o rhi/qrhigles2.cpp
$BUILD_PREFIX/bin/x86_64-apple-darwin13.4.0-clang++ -c -include.pch/Qt5Gui_x86_64.pch/objective-c++_x86_64 -pipe -stdlib=libc++ -Oz -fomit-frame-pointer -fdata-sections -fvisibility=hidden -std=c++1y -fapplication-extension  -arch x86_64 -isysroot /Applications/Xcode_11.3.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk -mmacosx-version-min=10.13 -fvisibility=hidden -fvisibility-inlines-hidden -fno-exceptions -Wall -Wextra -Winconsistent-missing-override -Wobjc-interface-ivars -Wobjc-method-access -Wobjc-multiple-method-names -Werror=unguarded-availability -Werror=unguarded-availability-new -Werror=unsupported-availability-guard -fPIC -DQT_NO_LINKED_LIST -DQT_NO_JAVA_STYLE_ITERATORS -DQT_NO_USING_NAMESPACE -DQT_NO_FOREACH -DENABLE_PIXMAN_DRAWHELPERS -DQT_NO_NARROWING_CONVERSIONS_IN_CONNECT -DQT_BUILD_GUI_LIB -DQT_BUILDING_QT -DQT_NO_CAST_TO_ASCII -DQT_ASCII_CAST_WARNINGS -DQT_MOC_COMPAT -DQT_USE_QSTRINGBUILDER -DQT_DEPRECATED_WARNINGS -DQT_DISABLE_DEPRECATED_BEFORE=0x050000 -DQT_DEPRECATED_WARNINGS_SINCE=0x060000 -DQT_NO_EXCEPTIONS -D_LARGEFILE64_SOURCE -D_LARGEFILE_SOURCE -DQT_NO_DEBUG -DQT_CORE_LIB -I. -I../3rdparty/md4c -I../3rdparty/VulkanMemoryAllocator -I../../include -I../../include/QtGui -I../../include/QtGui/5.14.1 -I../../include/QtGui/5.14.1/QtGui -I.tracegen -I/System/Library/Frameworks/OpenGL.framework/Headers -I/System/Library/Frameworks/AGL.framework/Headers/ -I../../include/QtCore/5.14.1 -I../../include/QtCore/5.14.1/QtCore -I../../include/QtCore -I.moc -I$PREFIX/include -I../../mkspecs/macx-clang -o .obj/qrhimetal.o rhi/qrhimetal.mm
rhi/qrhimetal.mm:219:44: error: unknown type name 'MTLCaptureManager'
    API_AVAILABLE(macos(10.13), ios(11.0)) MTLCaptureManager *captureMgr;
                                           ^
                         ^~~~~~~~
rhi/qrhimetal.mm:2941:20: error: use of undeclared identifier 'MTLVertexFormatUCharNormalized'
            return MTLVertexFormatUCharNormalized;
                   ^
rhi/qrhimetal.mm:3643:22: error: property 'displaySyncEnabled' not found on object of type 'CAMetalLayer *'
            d->layer.displaySyncEnabled = NO;
                     ^
4 warnings and 6 errors generated.
```

### CODE REVIEW

**Comment by m-kuhn** [comment]
@conda-forge-admin please rerender

**Comment by m-kuhn** [comment]
I'm running into different problems for each platform:

 - Linux: troubles installing Python 2 - I guess someone with conda knowledge can help
 - macOS: Some objective C code issues. Not much experience on my end, the condition macos 10.13 for MTLCaptureManager seems to be fine.
- Windows: Builder running out of disk space

Any inputs on how to deal with this appreciated.

File: .azure-pipelines/azure-pipelines-win.yml [hunk 1]
```diff
@@ -10,9 +10,9 @@ jobs:
   strategy:
     maxParallel: 4
     matrix:
-      win_cxx_compilervs2015vc14:
-        CONFIG: win_cxx_compilervs2015vc14
-        CONDA_BLD_PATH: C:\\bld\\
+      win_:
+        CONFIG: win_
+        CONDA_BLD_PATH: D:\\bld\\
```
Comment (gillins) [inline .azure-pipelines/azure-pipelines-win.yml]: @m-kuhn  You can change this back to `C:` to get more space (we have to do this for `qgis`). Next re-render will reset it however. You may still run out of time...

Comment (m-kuhn) [reply .azure-pipelines/azure-pipelines-win.yml]:
```suggestion
        CONDA_BLD_PATH: C:\\bld\\
```

**Comment by m-kuhn** [comment]
superseded by https://github.com/conda-forge/qt-feedstock/pull/156

**Pull request closed without merging**